ocaml/idl/datamodel.ml

```diff
@@ -9923,6 +9950,7 @@ let all_system =
     pvs_cache_storage;
     feature;
     sdn_controller;
+    vdi_nbd_server_info;
```

Comment:
The docstring of the `all_system` variable says:
> All the objects in the system in order they will appear in documentation

So in my opinion it would be better if we put `vdi_nbd_server_info` after `vdi`, because they are very strongly related.

Reply:
Ah, good point.